logsearch-jobs.yml

```diff
@@ -76,16 +82,12 @@ instance_groups:
   - name: curator
     release: logsearch
     # nil the link and use the colocated instance when https://github.com/cloudfoundry-community/logsearch-boshrelease/pull/90 is merged
```

Comment:
we should probably nuke this comment now, right?

Reply:
will do